### PR TITLE
Add WebProxy to ServiceFabric

### PR DESCRIPTION
**Pull request by Kahbazi**

### CODE REVIEW

File: src/ReverseProxy.ServiceFabric/ServiceDiscovery/Util/LabelsParser.cs [hunk 1]
```diff
@@ -317,8 +317,13 @@ internal static Cluster BuildCluster(Uri serviceName, Dictionary<string, string>
                     EnableMultipleHttp2Connections = GetLabel<bool?>(labels, "YARP.Backend.HttpClient.EnableMultipleHttp2Connections", null),
                     RequestHeaderEncoding = !string.IsNullOrEmpty(requestHeaderEncodingLabel) ? Encoding.GetEncoding(requestHeaderEncodingLabel) : null,
 #endif
+                    WebProxy = new WebProxyOptions
```
Comment (Tratcher) [inline src/ReverseProxy.ServiceFabric/ServiceDiscovery/Util/LabelsParser.cs:320]: https://github.com/microsoft/reverse-proxy/pull/879/files#diff-c36308ec27c72a71f1ddd3d9c179b2714b9483195035238efc8c13f7d6040af6R77-R83

The client factory uses the presence of this type to enable proxies, which might not be the best idea. Maybe that should check for a non-null Address field instead?

Comment (Kahbazi) [reply src/ReverseProxy.ServiceFabric/ServiceDiscovery/Util/LabelsParser.cs:320]: The non-null Address is being check here.
https://github.com/microsoft/reverse-proxy/pull/879/files#diff-c36308ec27c72a71f1ddd3d9c179b2714b9483195035238efc8c13f7d6040af6R97-R100
Is this enough?

Comment (Tratcher) [reply src/ReverseProxy.ServiceFabric/ServiceDiscovery/Util/LabelsParser.cs:320]: Good catch, nevermind.

**Review by Tratcher — APPROVED** [review]

**Comment by Tratcher** [comment]
Thanks

**Pull request merged**